boxoffice/views/admin_discount.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+import math
+from flask import jsonify
+from .. import app, lastuser
+from baseframe import _
+from coaster.views import load_models, render_with, requestargs
+from ..models import db
+from boxoffice.models import Organization, DiscountPolicy, DiscountCoupon, Price, CURRENCY_SYMBOL, CURRENCY
+from ..forms import DiscountPolicyForm, DiscountCouponForm, DiscountPriceForm, CouponBasedDiscountPolicyForm, AutomaticDiscountPolicyForm, PriceBasedDiscountPolicyForm
+from utils import xhr_only, date_time_format, api_error, api_success
+
+
+def jsonify_price(price):
+    if price:
+        return {
+            'amount': price.amount,
+            'start_at': date_time_format(price.start_at),
+            'end_at': date_time_format(price.end_at)
+        }
+    else:
+        return None
+
+
+def jsonify_discount_policy(policy):
+    return {
+        'id': policy.id,
+        'name': policy.name,
+        'title': policy.title,
+        'discount_type': "Automatic" if policy.is_automatic else "Coupon based",
+        'item_quantity_min': policy.item_quantity_min,
+        'percentage': policy.percentage,
+        'is_price_based': policy.is_price_based,
+        'discount_code_base': policy.discount_code_base,
+        'bulk_coupon_usage_limit': policy.bulk_coupon_usage_limit,
+        'discount': policy.percentage if not policy.is_price_based else '',
```

Comment:
@vidya-ram Is this attribute required? Since `percentage` and `is_price_based` are being returned above anyway.